packages/runtime/test/anonymous/tokens.test.ts

```diff
@@ -33,4 +34,12 @@ describe("Anonymous tokens", () => {
         const token = result.value;
         expect(token.content).toStrictEqual(uploadedToken.content);
     });
+
+    test("should catch a personalized token", async () => {
+        const uploadedPersonalizedToken = await uploadOwnToken(transportServices, runtimeService.address);
```

Comment:
wait - this is possible? You can personalize the token to yourself??

Reply:
yes, you can

Reply:
Does not really make sense, but nice for testing :D